plugins/rocks.xml.webhelp/xsl/topic.xsl

```diff
@@ -701,4 +709,25 @@
         <xsl:apply-templates select="*[not(contains(@class, ' topic/related-links '))]"/>
         <xsl:apply-templates select="*[contains(@class, ' topic/related-links ')]"/>
     </xsl:template>
+
+    <xsl:template match="*[contains(@class, ' topic/fig ')]" name="topic.fig">
+        <xsl:variable name="default-fig-class">
+            <xsl:apply-templates select="." mode="dita2html:get-default-fig-class"/>
+        </xsl:variable>
+        <xsl:apply-templates select="*[contains(@class, ' ditaot-d/ditaval-startprop ')]" mode="out-of-line"/>
+        <figure>
+            <xsl:if test="$default-fig-class != ''">
+                <xsl:attribute name="class" select="$default-fig-class"/>
+            </xsl:if>
+            <xsl:call-template name="commonattributes">
+                <xsl:with-param name="default-output-class" select="$default-fig-class"/>
+            </xsl:call-template>
+            <xsl:call-template name="setscale"/>
+            <xsl:call-template name="setidaname"/>
+            <xsl:apply-templates
+                    select="node() except *[contains(@class, ' topic/title ') or contains(@class, ' topic/desc ')]"/>
+        </figure>
+        <xsl:call-template name="place-fig-lbl"/>
```

Comment:
Looks like we get &lt;figcaption&gt; element in the html body. Should be &lt;div&gt; or &lt;p&gt;.